fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/SimpleSchemaPrinter.java

```diff
@@ -6,32 +6,3565 @@
+
+        }
+
+        @Override
+        public int getFetchDirection() throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public void setFetchSize(int rows) throws SQLException {
+
+        }
+
+        @Override
+        public int getFetchSize() throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public int getResultSetConcurrency() throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public int getResultSetType() throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public void addBatch(String sql) throws SQLException {
+
+        }
+
+        @Override
+        public void clearBatch() throws SQLException {
+
+        }
+
+        @Override
+        public int[] executeBatch() throws SQLException {
+
+            return null;
+        }
+
+        @Override
+        public Connection getConnection() throws SQLException {
+
+            return null;
+        }
+
+        @Override
+        public boolean getMoreResults(int current) throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public ResultSet getGeneratedKeys() throws SQLException {
+
+            return null;
+        }
+
+        @Override
+        public int executeUpdate(String sql, int autoGeneratedKeys) throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public int executeUpdate(String sql, int[] columnIndexes) throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public int executeUpdate(String sql, String[] columnNames) throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public boolean execute(String sql, int autoGeneratedKeys) throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public boolean execute(String sql, int[] columnIndexes) throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public boolean execute(String sql, String[] columnNames) throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public int getResultSetHoldability() throws SQLException {
+
+            return 0;
+        }
+
+        @Override
+        public boolean isClosed() throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public void setPoolable(boolean poolable) throws SQLException {
+
+        }
+
+        @Override
+        public boolean isPoolable() throws SQLException {
+
+            return false;
+        }
+
+        @Override
+        public void closeOnCompletion() throws SQLException {
+
+        }
+
+        @Override
+        public boolean isCloseOnCompletion() throws SQLException {
+
+            return false;
+        }
+
+    }
+
+    public class PrintResultSet implements java.sql.ResultSet {
```

Comment:
Is it really needed?

Reply:
Yes, so we get beyond the rs.next in the SELECT that checks for the version history table